debugger/oe-gdb

```diff
@@ -3,18 +3,10 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-# Get path of the oe-gdb script
-# See https://mywiki.wooledge.org/BashFAQ/028 for complexities involved
-# in determining location of a bash script. ${BASH_SOURCE}, though not perfect,
-# is an acceptable solution for oe-gdb.
-OE_GDB_DIR=$(dirname "${BASH_SOURCE[0]}")
-
-# Get the path to the debugger libraries relative to the oe-gdb path.
-# Normalize the path by cd-ing and doing a pwd -P.
-OE_GDB_LIB_DIR=$(cd "$OE_GDB_DIR/../lib/openenclave/debugger" || exit; pwd -P)
+echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
```

Comment:
Great that build/output/oe-gdb cannot be executed.

Reply:
That's actually an oversight on my part. The installation step took care of the permissions for me, and I did not check just the build output. Let's leave it as is?

Reply:
Yes. It is good that it cannot be executed.